expyfun/_version.py

```diff
@@ -1 +1,2 @@
 __version__ = '2.0.0.dev0'
+__fork__ = 'ilogue'
```

Comment:
Yes. The recommended config for development is to install expyfun using `python setup.py develop` (instead of `install`) which avoids creating this line (and also should avoid the virtual environment testing issue, because the version seen by the system will be whatever branch you happen to have checked out)